main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/application"
+	"github.com/josephroberts/edge-node-manager/database"
+	"github.com/josephroberts/edge-node-manager/device"
+	"github.com/josephroberts/edge-node-manager/radio"
+)
+
+func main() {
+	tidyOnExit()
+
+	database.Start()
+
+	radios := make(map[radio.SupportedRadio]radio.RadioInterface)
+	radios[radio.BLUETOOTH] = radio.Create(radio.BLUETOOTH)
+	//radios[radio.WIFI] = &radio.Wifi{}
+	//radios[radio.ZIGBEE] = &radio.Zigbee{}
+
+	applications := make(map[string]application.ApplicationInterface)
+	applications["resin"] = &application.Application{"resin", device.NRF51822, radios[radio.BLUETOOTH]}
+	//applications["applicationB"] = &application.Application{"applicationB", device.ESP8266, radios[radio.WIFI]}
+	//applications["applicationC"] = &application.Application{"applicationC", device.MICROBIT, radios[radio.BLUETOOTH]}
+
+	for {
+		for _, application := range applications {
+			application.Process()
+		}
+
+		time.Sleep(5 * time.Second)
```

Comment:
Again it's really nasty to have an uncommented arbitrary sleep in code like this. If we do need to sleep here, then a simple comment like 'delay between processing each set of applications to prevent 100% CPU usage' are really useful, a new maintainer doesn't know whether it's to hack around something, for a very specific purpose, is arbitrary etc.

Also I don't like the hard-coding, would make more sense to have this as either a constant in the config package or perhaps something configurable somewhere.